plugins/modules/device_configs_backup_workflow_manager.py

```diff
@@ -394,6 +398,11 @@ class Device_configs_backup(DnacBase):
     def __init__(self, module):
         super().__init__(module)
         self.skipped_devices_list = []
+        self.payload = module.params
+        self.keymap = {}
+        self.dnac_version = int(self.payload.get(
```

Comment:
setting dnac version is already done in dnac.py.. Can you check?

https://github.com/madhansansel/dnacenter-ansible/blob/main/plugins/module_utils/dnac.py
        self.payload = module.params
        self.dnac_version = int(self.payload.get("dnac_version").replace(".", ""))
        # Dictionary to store multiple versions for easy maintenance and scalability
        # To add a new version, simply update the 'dnac_versions' dictionary with the new version string as the key
        # and the corresponding version number as the value.
        self.dnac_versions = {
            "2.3.5.3": 2353,
            "2.3.7.6": 2376,
            "2.2.3.3": 2233
            # Add new versions here, e.g., "2.4.0.0": 2400
        }

        # Dynamically create variables based on dictionary keys
        for version_key, version_value in self.dnac_versions.items():
            setattr(self, "version_" + version_key.replace(".", "_"), version_value)

Reply:
Removed